esphome/components/i2c_device/__init__.py

```diff
@@ -0,0 +1,24 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import i2c
+from esphome.const import CONF_ID
+
+DEPENDENCIES = ["i2c"]
+CODEOWNERS = ["@gabest11"]
+
+MULTI_CONF = True
+i2c_device_ns = cg.esphome_ns.namespace("i2c_device")
+
+i2c_device = i2c_device_ns.class_("I2CDeviceComponent", cg.Component, i2c.I2CDevice)
```

Comment:
Variable names should always match the c++ class name
```suggestion
I2CDeviceComponent = i2c_device_ns.class_("I2CDeviceComponent", cg.Component, i2c.I2CDevice)
```